modules/bundle/custom_component/README.md

```diff
@@ -0,0 +1,15 @@
+# Description
+This example mainly shows a typical use case that brings customized python component(like: transform, network, metrics, etc.) in the `train` config.
```

Comment:
```suggestion
This example mainly shows a typical use case that brings customized python components (such as transform, network, metrics) in a configuration-based workflow.
```

Reply:
updated in the latest commit.

Thanks.